packages/contracts/test/voting/token-voting.ts

```diff
@@ -875,6 +893,14 @@ describe('TokenVoting', function () {
         ).to.equal(id);
       });
 
+      it('reverts on voting None', async () => {
+        await advanceIntoVoteTime(startDate, endDate);
```

Comment:
The same here as above.

Reply:
Added in 0fed43072dcd874cd9b4d43a70e0a28bc95a39e0